### PR TITLE
Add Script Commands: emoji

### PR DESCRIPTION
**Pull request by crstauf**

### CODE REVIEW

**Review by PitNikola — CHANGES_REQUESTED** [review]
As discussed in Slack and per [new section in guidelines about dependencies](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-dependencies), let's add a comment section explicitly stating the dependency and installation instruction.

**Comment by crstauf** [comment]
@PitNikola Added dependency comments.

File: commands/communication/emojis/emoji-copy.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Dependency: emoj (https://github.com/sindresorhus/emoj)
+# Install via npm: `npm install --global emoj`
+
+# @raycast.schemaVersion 1
+# @raycast.title Copy emoji
```
Comment (thomaspaulmann) [inline commands/communication/emojis/emoji-copy.sh:7]: Not sure if `Copy emoji` is a great name for this command. I couldn't tell what it does. How about this title?

```suggestion
# @raycast.title Search Emoji From Clipboard
```

File: commands/communication/emojis/emoji-copy.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.mode compact
```
Comment (thomaspaulmann) [inline commands/communication/emojis/emoji-copy.sh:8]: 👍

File: commands/communication/emojis/emojis-search.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Dependency: emoj (https://github.com/sindresorhus/emoj)
+# Install via npm: `npm install --global emoj`
+
+# @raycast.schemaVersion 1
+# @raycast.title Get emojis
```
Comment (thomaspaulmann) [inline commands/communication/emojis/emojis-search.sh:7]:
```suggestion
# @raycast.title Search Related Emojis from Clipboard
```

File: commands/communication/emojis/emojis-search.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# @raycast.mode fullOutput
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Search emojis for clipboard value.
+# @raycast.needsConfirmation true
```
Comment (thomaspaulmann) [inline commands/communication/emojis/emojis-search.sh:12]: Lets remove the `needsConfirmation`.

File: commands/communication/emojis/emoji-copy.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Copy first emoji found from clipboard value.
+# @raycast.packageName Communication
+# @raycast.needsConfirmation true
```
Comment (thomaspaulmann) [inline commands/communication/emojis/emoji-copy.sh:13]: I don't think you need a confirmation here.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Lets give the commands somewhat more descriptive titles. Otherwise they appear very general and are not easy to understand what they do.

File: commands/communication/emojis/emoji-copy.sh (hunk 1 above)
Comment (crstauf) [reply commands/communication/emojis/emoji-copy.sh:7]: This command searches for an emoji based on text in clipboard, and copies the first result to the clipboard. I'd like to give some indication to the user that the first result is copied.

File: commands/communication/emojis/emojis-search.sh (hunk 4 above)
Comment (crstauf) [reply commands/communication/emojis/emojis-search.sh:12]: As mentioned elsewhere, to me it's preferable to do a check to the user before the script accesses their clipboard, to confirm that they've the correct content there.

File: commands/communication/emojis/emoji-copy.sh (hunk 5 above)
Comment (crstauf) [reply commands/communication/emojis/emoji-copy.sh:13]: As mentioned elsewhere, to me it's preferable to do a check to the user before the script accesses their clipboard, to confirm that they've the correct content there.

Comment (thomaspaulmann) [reply commands/communication/emojis/emoji-copy.sh:13]: I'm not really convinced that this is a big issue. All commands are transparent and when you install them should be treated as safe. The confirmation dialog should be only used for irreversible actions, e.g. deleting files. I'll accept that one but I would please you to remove it from almost all commands to fit into the user experience of the rest of Raycast.

**Review by thomaspaulmann — APPROVED** [review]

**Pull request merged**